assets/syntaxes/Packages/Git Formats/Git Attributes.sublime-syntax

```diff
@@ -70,7 +70,7 @@ contexts:
 
   attributes-list:
     - meta_content_scope: meta.attributes-list.git.attributes
-    # attriubte list ends with line
+    # attribute list ends with line
```

Comment:
These also appear in the original:
https://github.com/sublimehq/Packages/blob/c7cd5a7dc3eb34b5746b2f87c5ca4967574c9e40/Git%20Formats/Git%20Attributes.sublime-syntax#L78